### DISCUSSION TITLE
Unfortunately, we weren't able to approve your application.

### DISCUSSION THREAD

**Discussion by YTHABIB** [community/community#116736] [GitHub Education]
### Hi everyone,

Unfortunately, we weren't able to approve your application.

**Comment by AidenJamesYt** [comment]
Could you please elaborate on what the issue is?

**Comment by ghostinhershell** [comment]
Hi @YTHABIB, we're happy you're here! You are more likely to get a useful response if you are explicit when describing the issue you're running into here, giving a few more details might help someone give you a nudge in the right direction.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open. 🙂
